### ISSUE TITLE
Swiftfomat removing numeric literals in parentheses

### ISSUE THREAD

**Issue by tkafka** [nicklockwood/SwiftFormat#1121]
Hello, I was using swiftformat to format https://github.com/kanchudeep/SunMoonCalculator/blob/master/SunMoonCalculator.swift and run into this issue of swiftformat deleting a number in parentheses:

before ([link to source](https://github.com/kanchudeep/SunMoonCalculator/blob/master/SunMoonCalculator.swift#L539)):
```swift
let eps:Double = (23.43929).toRadians
```

after swiftformat:
```swift
let eps: Double = .toRadians
```

**Comment by nicklockwood** [comment]
Thanks for reporting this, I'll get it fixed asap. For now the workaround is to disable the `redundantType` rule:

```
--disable redundantType
```

**Comment by tkafka** [comment]
Thank you for a swift response!

**Comment by nicklockwood** [comment]
@tkafka fixed in 0.49.2
